### ISSUE TITLE
Materials upload thumbnails lack descriptive alt text

### ISSUE THREAD

**Issue by vbrown608** [EFForg/sec#366]
The alt text is being autogenerated from the filename. Depending on how they're displayed in context, it's possible that they will just need an empty alt attribute. Repurposing the description field is also a possibility.

:candy: :candy: :candy:
